src/Umbraco.Core/Configuration/Models/ContentImagingSettings.cs

```diff
@@ -23,7 +23,7 @@ public class ContentImagingSettings
             }
         };
 
-        internal const string StaticImageFileTypes = "jpeg,jpg,gif,bmp,png,tiff,tif";
+        internal const string StaticImageFileTypes = "jpeg,jpg,gif,bmp,png,tiff,tif,webp";
```

Comment:
This setting doesn't seem to be used: the supported image file types are now retrieved from the `IImageUrlGenerator.SupportedImageFileTypes` property and the `ImageSharpImageUrlGenerator` implementation dynamically retrieves the supported ones from the ImageSharp configuration instead.

These image file types are then used to for more then just generating image URLs though (e.g. whether to extract image dimensions to auto-fill properties, disallow file uploads that are not images and automatically changing the media type from `File` to `Image`).

I suggest keeping this setting and create a separate PR to fix the mixed usage, so:
- `ContentImagingSettings.ImageFileTypes` is used to determine whether the uploaded file is a valid image and if it can automatically change the media type to `Image`;
- `IImageUrlGenerator.SupportedImageFileTypes` is used whether it can generate an image URL for a specified file;
- A new `IImageDimensionExtractor.SupportedImageFileTypes` property is added that exposes whether it can get dimensions for a specified file (and have the `ImageSharpDimensionExtractor` populate this the same way as the `ImageSharpImageUrlGenerator` currently does).